application/libraries/SurveyPatch/Op/Op.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace LimeSurvey\SurveyPatch\Op;
+
+class Op implements OpInterface
```

Comment:
This is part of JSON patch blob spec or?

Reply:
Yeah. I started working on the patch functionality in this branch when I realised it was quite a chunk of work and created a separate task / branch.

Reply:
If nothing else it ties my work to the task in Jira. When you ask for a PR for the task I cant give you one if I don't have a branch.

If it was down to me, we would work directly on the develop branch when possible. I dont think thats going to happen though. Certainly not in the near future.

Reply:
Is that a yes or a no? :)

Reply:
Working directly on branches have been tried and failed hard, so that would be a long discussion, yes.

Reply:
For sure. Trunk based development requires good test coverage and loosely coupled components. Its not an option for us at this time. Its not worth discussing.

Reply:
> Is that a yes or a no? :)

Yes.

I don't have a choice its required for for me to create a PR.